docs/csharp/tutorials/nullable-reference-types.md

```diff
@@ -38,9 +38,6 @@ Create a new console application either in Visual Studio or from the command lin
 <Nullable>enable</Nullable>
 ```
 
-> [!IMPORTANT]
-> The `Nullable` element was previously named `NullableContextOptions`. The rename ships with Visual Studio 2019, 16.2-p1. The .NET Core SDK 3.0.100-preview5-011568 does not have this change. If you are using the .NET Core CLI, you'll need to use `NullableContextOptions` until the next preview is available.
-
 > [!NOTE]
 > When C# 8 is released (not in preview mode), the `Nullable` element will be added by new project templates. Until then, you'll need to add it manually.
```

Comment:
And what about this one? Can it go as well?

Reply:
I think yes, C# is already released and not in preview mode. I'm going to remove that note too.